ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandDeduplicationIT.scala

```diff
@@ -22,4 +22,8 @@ final class CommandDeduplicationIT(timeoutScaleFactor: Double, ledgerTimeInterva
   }
 
   override def testNamingPrefix: String = "ParticipantCommandDeduplication"
+
+  /** Assertions for append-only schema are important for KV ledgers
+    */
```

Comment:
```suggestion
  /** Enables additional assertions available when running with the append-only schema.
    */
```

Reply:
i.e., I'd avoid a ledger-specific reference in the general test logic. Maybe you can add a comment about KV in the KV subclass.

Reply:
Yes, I will address that in a follow-up PR to clean-up the tests